CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ MongoDB plugin for IntelliJ IDEA.
 ## [Unreleased]
 
 ### Added
+* [INTELLIJ-11](https://jira.mongodb.org/browse/INTELLIJ-11): Flush pending analytics events before closing the IDE.
```

Comment:
Is this the general format we'll be using for changelog management? I'm a bit worried that this will lead to merge conflicts once there's more than one person working on this product 🙂

Reply:
Merge conflicts in the CHANGELOG file you mean? Yeah I mean, that's an option, but I don't think that should be really troublesome. It should be a trivial merge conflict to fix.

This is the recommended way (from JetBrains at least) to keep track of the plugin changelog, as they already have a plugin that handles everything integrated (https://github.com/JetBrains/gradle-changelog-plugin).

Reply:
Alright – I think we've very intentionally tried to avoid this for Compass and mongosh, but if we're being diligent here and are okay with having to resolve merge conflicts, this is all fine by me 🙂